spiceaidocs/docs/components/data-connectors/clickhouse.md

```diff
@@ -16,20 +14,50 @@ datasets:
 
 ## Configuration
 
+### `from`
+
+The `from` field for the Clickhouse connector takes the form of `from:path_to_my_dataset` where `path_to_my_dataset` is the path to the Dataset within Clickhouse. In the example above it would be `path.to.my_dataset`.
```

Comment:
Clickhouse only supports a two-level path scheme, of the format `database_name.table_name`. So specifying a three part path here is incorrect. This documentation of `from` should explain how the logical grouping model of clickhouse maps to the path selector. i.e. `db.table` or just `table` for the `default` db

See https://clickhouse.com/docs/en/guides/creating-tables

Reply:
Fixed